### ISSUE TITLE
xbps-install: resume an interrupted download fails

### ISSUE THREAD

**Issue by gt7-void** [void-linux/xbps#212]
I have a spotty internet connection, which means that large package downloads are often interrupted. I noticed that every time an interrupted download is resumed, there's a signature error at the end.

I can reproduce this consistently in a different box with reliable internet connection, by interrupting a download with `^C`. For instance:
```
$ sudo xbps-install -yD openssh

Name    Action    Version           New version            Download size
openssh update    8.1p1_1           8.1p1_2                705KB 

Size to download:              705KB
Size required on disk:         705KB
Space available on disk:       252GB


[*] Downloading packages
^Censsh-8.1p1_2.x86_64.xbps: [705KB 21%] 445KB/s ETA: 00m03s
$ sudo xbps-install -yD openssh

Name    Action    Version           New version            Download size
openssh update    8.1p1_1           8.1p1_2                705KB 

Size to download:              705KB
Size required on disk:         705KB
Space available on disk:       252GB


[*] Downloading packages
openssh-8.1p1_2.x86_64.xbps: 501KB [avg rate: 439KB/s]
openssh-8.1p1_2: verifying RSA signature...
ERROR: openssh-8.1p1_2: the RSA signature is not valid!
ERROR: openssh-8.1p1_2: removed pkg archive and its signature.
Transaction failed! see above for errors.
```
This happens *every* time a download is interrupted.

**Comment by xtraeme** [comment]
I believe this is fixed in master.

**Comment by gt7-void** [comment]
Indeed, I confirm it was fixed by 013177ce.

That was quick, thanks  :-)
